tests/test_parse_enex.py

```
@@ -0,0 +1,472 @@
+    assert "- two" in out
+
+
+def test_enml_blockquote() -> None:
+    enml = "<blockquote>引用</blockquote>"
+    assert "> 引用" in enml_to_markdown(enml)
+
+
+def test_enml_entity_decoding() -> None:
+    enml = "<p>A&nbsp;B&amp;C&lt;D</p>"
+    out = enml_to_markdown(enml)
+    assert "A" in out and "B&C<D" in out
```

Ruff (0.15.12)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)